packages/fuels-signers/src/provider.rs

```diff
@@ -557,15 +560,15 @@ mod tests {
     #[test]
     fn test_calculate_tx_cost() -> Result<(), Error> {
         let transaction_cost =
-            Provider::calculate_tx_cost(Transaction::default(), 10_000, 10_000, 0, 0, 270)?;
+            Provider::calculate_tx_cost(Transaction::default(), 10_000, 10_000, 42, 42, 270)?;
 
-        assert_eq!(transaction_cost.min_gas_price, 0);
-        assert_eq!(transaction_cost.min_byte_price, 0);
+        assert_eq!(transaction_cost.min_gas_price, 42);
+        assert_eq!(transaction_cost.min_byte_price, 42);
         assert_eq!(transaction_cost.gas_price, 10_000);
         assert_eq!(transaction_cost.byte_price, 10_000);
         assert_eq!(transaction_cost.gas_used, 270);
         assert_eq!(transaction_cost.byte_size, 120);
-        assert_eq!(transaction_cost.fee, 0.0039);
+        assert_eq!(transaction_cost.total_fee, 0.0039);
```

Comment:
`gas_used` and `total_fee`, since they are computed, should be be computed alongside and displayed in the test, so we can check that the computation is the one we expect.